.travis.yml

```diff
@@ -14,10 +14,19 @@ php:
   - nightly
 
 matrix:
+  include:
+    - name: "Windows"
+      os: windows
+      language: shell # no built-in php support
+      before_install:
```

Comment:
Looks like you may want to also update or overwrite the `install` and/or `script` instructions for Windows :+1:

Reply:
yeah I missed that, will fix it later tonight 👍